llvm/test/CodeGen/PowerPC/lit.local.cfg

```diff
@@ -1,4 +1,22 @@
 if not "PowerPC" in config.root.targets:
     config.unsupported = True
 
+import subprocess
+
 config.suffixes.add(".py")
+
+def get_revision(repo_path):
+    cmd = ['git', '-C', repo_path, 'rev-parse', 'HEAD']
+    try:
+        return subprocess.run(cmd, stdout=subprocess.PIPE, check=True).stdout.decode('ascii')
+    except subprocess.CalledProcessError:
+        print("An error occured retrieving the git revision")
+        raise
```

Comment:
nit: do you really want to just propagate this exception? Seems like it might be better to just disable the test if we can't run `git`, rather than abort `lit` (my gut feel is this will break some weird buildbot config somewhere)